src/execution_plan/ops/op_merge.c

```diff
@@ -111,10 +103,8 @@ static void _CommitEdges(OpMerge *op, Record r) {
 
         // TODO: For the timebeing if edge has multiple relationship types, use the first one.
         // ()-[:A|B]->()
```

Comment:
Merge clauses that specify multiple relation types are not allowed (because a relation must have exactly 1 type). Sample Neo handling:
```
Neo.ClientError.Statement.SyntaxError: A single relationship type must be specified for MERGE (line 1, column 39 (offset: 38))
"MERGE (a:Actor {name: 'Nicolas Cage'})-[:ACTS_IN|DIRECTED]->(b:banana)"`
```

Reply:
👍